bigframes/core/blocks.py

```diff
@@ -498,7 +498,7 @@ def to_pandas(
         sampling_method: Optional[str] = None,
         random_state: Optional[int] = None,
         *,
-        ordered: Optional[bool] = None,
+        ordered: bool = True,
```

Comment:
Does this mean total ordering now? That is to say, can ordered = False support partial ordering?

Let's add this info to the docstring.

Reply:
Added docstring and indicate that ordered=True will sort rows, but whether that is total or not depends on session setting.

ordered=False will always be completely unordered.